citi-onboarding/celeiro-dos-bambas
# Hotfix/navbar links

## What I did

- Created the anchor tag links in Navbar; added smooth scroll and fixed the scroll padding to it; fixed the padding between the Banner section and the Navbar and, finally, adjusted the impact section to different screen sizes(
vertically and horizontally).

## How to test

1. In terminal:
   1. `git checkout main`
   1. `git pull`
   1. `git checkout hotfix/navbarLinks`
   1. `git pull`
   1. `yarn run develop` in `server/` folder
   1. `yarn run start` in `client/` folder
 2. Go to [http://localhost:3000/](http://localhost:3000/)
 3. Check if the adjustments are as expected. To test the screen size adjustments, check the site with different browser screen sizes(vertically and horizontally).
